### PR TITLE
Combine Individuals and Sample sets pages

### PR DESCRIPTION
**Pull request by almanilsson**
This PR moves the contents of the sample sets page into the individuals page. It is dependent on the changes made in PR #100 and #105. 

image

### CODE REVIEW

**Comment by almanilsson** [comment]
Note: currently you have to click update in "Batch reassign individuals" after adding a new sample in order to see that index as an alternative. Should fix that.

**Comment by hyanwong** [comment]
Screen shots look lovely. I'm looking forward to seeing this in action.

File: src/tseda/datastore.py [hunk 1]
```diff
@@ -78,7 +78,8 @@ class SampleSetsTable(Viewer):
     }
 
     create_sample_set_textinput = param.String(
-        doc="Enter name of new sample set. Press Enter (⏎) to create.",
+        doc="""Enter name of new sample set. Press Enter (⏎) to create.
+            If the new sample set does not immediately show up, refresh the page""",
```
Comment (almanilsson) [inline src/tseda/datastore.py:82]: Added issue #114 about this

Comment (almanilsson) [reply src/tseda/datastore.py]: added a better temporary solution in https://github.com/hannamlmv/tseda/pull/107/commits/77aadf40d84491564a192e37e873fb2b2804576e

**Review by tforest — COMMENTED** [review]
Tried this on my side. To me that makes more sens after that merge, thank you! :+1: 

 I think your test is failing because the "Map options" is collapsed by default, or maybe it takes to long to be displayed, so maybe adding a timeout would fix it?

**Review by percyfal — APPROVED** [review]
LGTM

**Pull request merged**